### PR TITLE
Support MySQL Part 2

### PR DESCRIPTION
**Pull request by HaitaoDeng**

### CODE REVIEW

**Review by Jack-Jiang — APPROVED** [review]

File: lib/ui/mysql/widget/mysql_tables.dart [hunk 1]
```diff
@@ -23,10 +23,13 @@ import 'package:paas_dashboard_flutter/route/page_route_const.dart';
 import 'package:paas_dashboard_flutter/ui/component/searchable_title.dart';
 import 'package:paas_dashboard_flutter/ui/util/exception_util.dart';
 import 'package:paas_dashboard_flutter/ui/util/spinner_util.dart';
+import 'package:paas_dashboard_flutter/vm/mysql/mysql_sql_query_view_model.dart';
 import 'package:paas_dashboard_flutter/vm/mysql/mysql_table_data_view_model.dart';
 import 'package:paas_dashboard_flutter/vm/mysql/mysql_table_view_model.dart';
 import 'package:provider/provider.dart';
 
+import 'mysql_sql_query.dart';
```
Comment (goflutterjava) [inline lib/ui/mysql/widget/mysql_tables.dart:31]: can use relative import?

File: lib/vm/mysql/mysql_table_index_view_model.dart [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
```
Comment (goflutterjava) [inline lib/vm/mysql/mysql_table_index_view_model.dart:28]: no need double license

**Review by fu-turer — APPROVED** [review]

File: lib/ui/mysql/widget/mysql_tables.dart (hunk 1 above)
Comment (HaitaoDeng) [reply lib/ui/mysql/widget/mysql_tables.dart]: OK

File: lib/vm/mysql/mysql_table_index_view_model.dart (hunk 2 above)
Comment (HaitaoDeng) [reply lib/vm/mysql/mysql_table_index_view_model.dart]: OK

**Review by goflutterjava — APPROVED** [review]

**Pull request merged**